public/javascript/getWeather.js

```diff
@@ -65,9 +78,9 @@ module.exports = function getWeather(api, threadID, body) {
                     } else {
                         emoji = '';
                     }
-
+                    var displayType = currentDType === 'F' ? '\xB0F\n' : '\xB0C\n';
```

Comment:
I think it can be a const as well right? Considering we aren't changing the display degree type afterwards.